src/views/Drawer/DrawerView.js

```diff
@@ -114,9 +114,17 @@ export default class DrawerView<T: *> extends PureComponent<void, Props, void> {
 
   render() {
     const DrawerScreen = this.props.router.getComponentForRouteName('DrawerClose');
+
+    const screenNavigation = addNavigationHelpers({
+      state: this._screenNavigationProp.state,
+      dispatch: this._screenNavigationProp.dispatch,
+    });
```

Comment:
@satya164 Not 100% sure what's happening here, but this seems to work. Is there a better way to convert the flow type?